config/deploy.rb

```diff
@@ -0,0 +1,44 @@
+# config valid for current version and patch releases of Capistrano
+lock "~> 3.10.1"
+
+# set :application, "my_app_name"
+# set :repo_url, "git@example.com:me/my_repo.git"
+
+set :application, "heroes_team_app"
+set :repo_url, "git@github.com:Ruby-IF081/heroes_team_app.git"
+
+# Default branch is :master
+ask :branch, :master # `git rev-parse --abbrev-ref HEAD`.chomp
```

Comment:
it is better to set branch in production.rb

Reply:
done